packages/date-picker/src/vaadin-date-picker-mixin.js

```diff
@@ -313,6 +324,23 @@ export const DatePickerMixin = (subclass) =>
       };
     }
 
+    static calculateDateBasedOnReferenceDate(refDate, year, month, date) {
```

Comment:
Consider moving this function to `vaadin-date-picker-helper.js` where other helpers are placed.
Then you could import it from there in the Flow component connector.